vendor/engines/secure_rooms/app/controllers/secure_rooms_api/scans_controller.rb

```diff
@@ -26,6 +28,10 @@ def load_user_and_reader
       )
     end
 
+    def render_not_found(exception)
+      render json: { error: exception.message }, status: 404
```

Comment:
yeah looks like it's not readily available and we'd have to dig out the right id since we're finding these by different-named attributes like `#card_number`